code/modules/research/xenoarchaeology/finds/finds_fossils.dm

```diff
@@ -46,16 +46,19 @@
 /obj/skeleton
 	name = "Incomplete skeleton"
 	icon = 'icons/obj/xenoarchaeology.dmi'
-	icon_state = "uskel"
 	desc = "Incomplete skeleton."
 	var/bnum = 1
+	var/skeleton_type = "beast"
+	var/list/skeleton_types = list("biped", "beast")
 	var/breq
 	var/bstate = 0
 	var/plaque_contents = "Unnamed alien creature"
 
 /obj/skeleton/New()
 	src.breq = rand(6)+3
 	src.desc = "An incomplete skeleton, looks like it could use [src.breq-src.bnum] more bones."
+	skeleton_type = pick(skeleton_types)
+	icon_state = "uskel_[skeleton_type]"
```

Comment:
I'm pretty you've missed porting some sprites here.

Reply:
wym